resources/js/src/app/components/itemList/filter/ItemRangeSlider.vue

```diff
@@ -82,7 +82,8 @@ export default defineComponent({
       return 'fl-range-slider-' + props.facet.id
           .replace(/\W/g, '-')
           .replace(/-+/, '-')
-          .replace(/-$/, '');
+          .replace(/-$/, '')
+          + Math.random().toString(36).substr(2, 9);
```

Comment:
I am a bit worried about the random string within the `computed` property. Because by the nature of `computed`, it will update the property every tick.
It would maybe be saver to set this random string as a constant at the beginning of the setup and just use it in this line.